privacyidea/lib/auditmodules/loggeraudit.py

```diff
@@ -69,8 +69,10 @@ class Audit(AuditBase):
     .. note:: This audit module does not provide a *Read* capability.
     """
 
-    def __init__(self, config=None):
-        super(Audit, self).__init__(config)
+    def __init__(self, config=None, startdate=None):
+        super(Audit, self).__init__(config, startdate)
+        # overwrite the startdate from the superclass with the isoformat
+        self.audit_data["startdate"] = startdate or datetime.utcnow().isoformat()
```

Comment:
We would need `startdate.isoformat()` here otherwise the `json.dumps()` fails.

Reply:
Will do.

Reply:
Done in 1da2c0a5